mobile/asa-go/src/components/profile/FuelSummary.tsx

```diff
@@ -0,0 +1,184 @@
+import { useEffect, useState } from "react";
+import { FireShape, FireZoneFuelStats } from "api/fbaAPI";
+import { Box, Tooltip, Typography } from "@mui/material";
+import { groupBy, isUndefined } from "lodash";
+import {
+  DataGridPro,
+  GridColDef,
+  GridColumnHeaderParams,
+  GridRenderCellParams,
+} from "@mui/x-data-grid-pro";
+import { styled } from "@mui/material/styles";
+import CriticalHours from "@/components/profile/CriticalHours";
+import FuelDistribution from "@/components/profile/FuelDistribution";
+
+export interface FuelTypeInfoSummary {
+  area: number;
+  criticalHoursStart?: number;
+  criticalHoursEnd?: number;
+  id: number;
+  code: string;
+  description: string;
+  percent?: number;
+  selected: boolean;
+}
+
+interface FuelSummaryProps {
+  fireZoneFuelStats: Record<number, FireZoneFuelStats[]>;
+  selectedFireZoneUnit: FireShape | undefined;
+}
+
+const StyledHeader = styled("div")({
+  whiteSpace: "normal",
+  wordWrap: "break-word",
+  textAlign: "center",
+  fontSize: "0.75rem",
+  fontWeight: "700",
+});
+
+// Column definitions for fire zone unit fuel summary table
+const columns: GridColDef[] = [
+  {
+    field: "code",
+    headerClassName: "fuel-summary-header",
+    headerName: "Fuel Type",
+    sortable: false,
+    minWidth: 120,
+    renderHeader: (params: GridColumnHeaderParams) => (
+      <StyledHeader>{params.colDef.headerName}</StyledHeader>
+    ),
+    renderCell: (params: GridRenderCellParams) => (
+      <Tooltip followCursor placement="right" title={params.row["description"]}>
+        <Typography sx={{ fontSize: "0.75rem", display: "flex", flexGrow: 1 }}>
```

Comment:
If we add `height: "100%", alignItems: "center"` to the Typography sx prop the text should be centered vertically.

Reply:
https://github.com/bcgov/wps/pull/4708/commits/f465787707ea56ae6c04e22b9d6cafc360fa1668